packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -301,6 +301,28 @@ governing permissions and limitations under the License.
   }
 }
 
+/* additional a11y support by moveing scrolling from content to dialog */
+@media screen and (max-height: 400px) {
+  .spectrum-Dialog .spectrum-Dialog-grid {
+    border-top-left-radius: var(--spectrum-dialog-border-radius);
+    border-top-right-radius: var(--spectrum-dialog-border-radius);
```

Comment:
why is this only for the small screen height case? could this just be on the top level outermost .spectrum-Dialog .spectrum-Dialog-grid?
What is it fixing though?

Reply:
This is fixing the rounded corners on the hero image when scrolling which only happens in the small screen scenario. I created a simplified example to replicate our issue. Applying rounded corners to the `spectrum-Dialog-grid` gave the hero rounded corners when scrolling.

Reply:
Ah, I see, I had to actually scroll to see it. It's easier to see in light mode as well, for anyone else checking